reactDashboard/src/components/HeatMap/Heatmap.css

```diff
@@ -78,38 +98,69 @@
 .heatmapScale .l-6 {
   background-color: #bd0026;
 }
+
+.darkmode .heatmapScale .l-0,
+.darkmode .heatmapScale .l-1,
+.darkmode .heatmapScale .l-2,
+.darkmode .heatmapScale .l-3 {
+  color: #020202;
+}
+
 .heatmapScale .l-7 {
   background-color: #020202;
   color: white;
 }
 
 .heatmapScale {
-  padding-top: 5px;
-  padding-bottom: 5px;
+  padding-top: 0;
+  padding-bottom: 0;
+  margin-bottom: 10px;
+  margin-left: 0;
+  border: 1px solid darkgrey;
 }
-.heatmapScale span {
-  padding: 5px 10px;
+
+.darkmode .heatmapScale {
+  border-color: #f2f2f2;
+  background-color: #f2f2f2;
 }
-.heatmapScale span.smallscale {
+
+.heatmapScale .row {
+  flex-wrap: nowrap;
+}
+
+.heatmapScale .col {
+  padding: 2px 15px;
+}
+
+.heatmapScale .col.smallscale {
   display: none;
 }
+
 @media all and (max-width: 810px) {
-  .heatmapScale span.smallscale {
+  .heatmapScale .col.smallscale {
     display: inline;
   }
-  .heatmapScale span.largescale {
+  .heatmapScale .col.largescale {
     display: none;
   }
 }
 
+.darkmode .heatmapScale {
+  color: #020202;
+}
+
+.darkmode .heatmapScale .l-7 {
+  color: white;
+}
+
 @media all and (max-width: 600px) {
   .heatmap {
     font-size: 80%;
   }
 }
 
 @media all and (max-width: 450px) {
```

Comment:
max-width should be increased from 450 to about 500px to avoid the heatbars being truncated

Reply:
✅ 😎

Reply:
changed to 500px

Reply:
Ok, there are still a number of widths where the heatmap table gains a horizontal scrollbar. We want to avoid that. Can the paddings and media query widths be adjusted so that the table always fits in the horizontal space available ?

Reply:
refactored